frontend/packages/component-library/src/video/index.ts

```diff
@@ -1,2 +1,5 @@
+// Add this to prevent a Next.js build error in Contentful
+'use client';
```

Comment:
I wonder if it's possible that this will break component for non-next.js consumers(e.g. dashboard)?

Also I wonder if we should add this to `frontend/apps/marketing/src/components/video/index.ts` or `frontend/packages/component-library/src/video/Video.tsx`  instead of this index/ts

Reply:
I'll try moving it to those files 🤞

Reply:
It throws errors on those two files 😣

Reply:
I just tried adding this on a studio.code.org page it works w/ the `use client` directive so I think this is ok?
<img width="1729" alt="Screenshot 2025-02-03 at 10 19 43 AM" src="https://github.com/user-attachments/assets/e04b1071-39a2-4f25-8ed0-6a3d583899dd" />

Reply:
This is OK to start with, but ideally we will want to avoid using `use client` as it prevents it from being server-side rendered. Could we open a task to remove this directive and log the error that is produced?

Reply:
Is there something that can be done on Contentful's side to prevent this error?